cargo-annotation/action.yml

```diff
@@ -20,8 +20,10 @@ runs:
         fi
 
         if [[ "${{ inputs.with-annotation }}" == "true" ]]; then
-          CMD="$CMD --message-format=json | ${{ github.action_path }}/annotation.sh"
+          CMD="$CMD --message-format=json"
         fi
 
-        eval "$CMD"
+        eval "$CMD -- -D warnings | ${{ github.action_path }}/annotation.sh"
```

Comment:
We shouldn't rely on the `-- -D warning` here as it will cause clippy to exit after the first warning.